ambiata-mafia.cabal

```diff
@@ -22,7 +22,7 @@ library
                     , attoparsec                      >= 0.12       && < 0.14
                     , base16-bytestring               == 0.1.*
                     , bytestring                      == 0.10.*
-                    , Cabal                           >= 1.22       && < 1.25
+                    , Cabal                           >= 2.0        && < 2.1
```

Comment:
This lower bound seems like it would stop use of Cabal 1.24.0.*?

Reply:
Yes, code currently doesn't compile against the old version of the library. Had to do this to avoid problems with `Setup.hs` and the fact that GHC 7.10 doesn't support version macros in that file.